blocks/columns/luxury-collection-template.css

```diff
@@ -0,0 +1,74 @@
+.luxury-collection .columns.block {
+    max-width: var(--normal-page-width);
+}
+
+.luxury-collection .columns.block h2 {
+    font: normal normal 300 50px/56px var(--font-family-gotham);
+    letter-spacing: 0;
+    margin-bottom: 25px;
+}
+
+.luxury-collection .columns.block h3 {
+    font: normal normal 700 16px/23px var(--font-family-gotham);
+    letter-spacing: 2.08px;
+    margin-bottom: 24px;
+}
+
+.luxury-collection .columns.block p{
+    font-size: 18px;
+    line-height: 24px;
+    letter-spacing: .9px;
+    margin-bottom: 20px;
+}
+
+.luxury-collection .columns > div {
+    align-items: baseline;
+}
+
+.luxury-collection .columns.block img {
+    max-width: 315px;
+    height: auto;
+    float: left;
```

Comment:
Will this assumption hold true for all lux columns with an image? Will it always float on the left?